src/components/common/Button.tsx

```
@@ -0,0 +1,64 @@
+  leftIcon,
+  rightIcon,
+  children,
+  className,
+  ...props
+}: ButtonProps) => {
+  const activeVariant = props.disabled ? "gray" : variant;
+
+  return (
+    <button
+      className={cn(
+        "inline-flex transition-colors cursor-pointer",
```

ESLint

[error] Replace `transition-colors·cursor-pointer` with `cursor-pointer·transition-colors`

(prettier/prettier)